examples/jsm/csm/Shader.d.ts

```diff
@@ -1,5 +1,8 @@
-declare namespace _default {
-    export const lights_fragment_begin: string;
-    export const lights_pars_begin: string;
+declare namespace Sharder {
```

Comment:
`Sharder`? Looks like a typo?

Reply:
> `Sharder`? Looks like a typo?

TBH, I think rename as the file name reasonable, if not , I will recover it.

Reply:
Read again, there is an `r` too much in the name.

Reply:
> Read again, there is an `r` too much in the name.

Sorry, you mean I should recover it?